index.html

```diff
@@ -19,14 +19,20 @@
     </div>
   </header>
   <section class="container">
-    <div class="contarinerLeft">
+    <div class="containerLeft">
+      <i class="fa-solid fa-chevron-left control up" id="up"></i>
+      <div class="leftInto" id="divFlags">
+        <ul class="principalList" id="flags">        
+        </ul>
+      </div>      
+      <i class="fa-solid fa-chevron-left control down" id="down"></i>
+    </div>
+    <div class="containerCenter">
       <textarea class="textArea" id="inputText" rows="20" cols="60" placeholder="Type your text..."></textarea>
       <div class="buttons">
-        <select id="flags" class="flag">
-        </select>
         <select class="fa gender-select" id="gender">
-          <option class="fa icon-gender mal" value="Male">👨‍🦱</option>
-          <option class="fa icon-gender fem" value="Female">👩</option>
+          <option class="fa icon-gender mal" value="male">👨‍🦱</option>
+          <option class="fa icon-gender fem" value="female">👩</option>
```

Comment:
These classes (fa icon-gender) are really neccessary?

mal and fem aren't good names for this classes.